dev/com.ibm.ws.jbatch.security/src/com/ibm/ws/jbatch/security/impl/WSBatchAuthServiceImpl.java

```diff
@@ -253,27 +337,62 @@ private long authorizedJobStopRestartByInstance(long instanceId)
     	return instanceId;
     }
     
+    private void authorizedWriteOperationAccessCheck(String submitter, long instanceId) 
+            throws NoSuchJobExecutionException, JobSecurityException {
+		Set<String> listOfGroupsForJobID = null;
+		Set<String> listOfGroupsForSubject = null;
+		boolean checkGroupSecurity = false;
+
+		if (isInAnyBatchRole()) {
+			if (this.isAdmin(runAsSubject())) {
+			} else if (this.isSubmitter(runAsSubject())) {
+				boolean submitterRole = this.isSubmitter(runAsSubject());
+				boolean isGroupAdmin = this.isGroupAdmin();
+				boolean isGroupMonitor = this.isGroupMonitor();
+				// if you don't also own the job you can't view it
+				if (batchSecurityHelper.getRunAsUser().equals(submitter)) {
+				} else if (isGroupAdmin) {
+					checkGroupSecurity = true;
+				}
+			} else if (this.isGroupAdmin()) {
+				checkGroupSecurity = true;
+			} else if (this.isMonitor(runAsSubject()) || this.isGroupMonitor()) {
```

Comment:
maybe just don't check... so change line 359 to `} else {`   if we somehow fail this else-inf check we're going to be authorized.

Reply:
Not that I can see how that happens..just being defensive.